build.gradle

```diff
@@ -3,9 +3,14 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
+            url 'https://repos.zeroturnaround.com/nexus/content/repositories/zt-public-releases'
```

Comment:
this is actually unnecessary... it comes from JRebel Android plugin (private beta)